package.json

```
@@ -1,17 +1,18 @@
 {
-  "name": "dip721",
-  "description": "This package provides the DIP-721 developer utility scripts",
-  "version": "0.0.1",
-  "main": "index.js",
-  "repository": "https://github.com/Psychedelic/DIP721.git",
-  "author": "Helder Oliveira <helder@fleek.co>",
-  "license": "MIT",
-  "scripts": {
-    "cap:init": "git submodule update --init --recursive",
-    "cap:start": "cd ./cap && dfx deploy ic-history-router && mkdir -p ../.temp && dfx canister id ic-history-router > ../.temp/ic-history-router-id",
-    "dip721:healthcheck": "./healthcheck.sh",
-    "reset": "dfx stop && rm -rf .dfx && rm -rf ./cap/.dfx && rm -rf ./.temp",
-    "dev:watch": "cargo watch -w ./nft/src -s 'yarn reset && dfx start --background && yes Y | SKIP_PROMPTS=1 yarn dip721:healthcheck'",
-    "test": "cargo test -- --show-output"
-  }
+	"name": "dip721",
+	"description": "This package provides the DIP-721 developer utility scripts",
+	"version": "0.0.1",
+	"main": "index.js",
+	"repository": "https://github.com/Psychedelic/DIP721.git",
+	"author": "Helder Oliveira <helder@fleek.co>",
+	"license": "MIT",
+	"scripts": {
+		"cap:init": "git submodule update --init --recursive",
+		"cap:start": "cd ./cap && dfx deploy ic-history-router && mkdir -p ../.temp && dfx canister id ic-history-router > ../.temp/ic-history-router-id",
+		"healthcheck": "./.scripts/healthcheck.sh",
+		"deploy": "./.scripts/deploy.sh",
```

Let's follow the original convention for the services and have the name as a prefix, as we do for the Cap service.

```suggestion
		"dip721:healthcheck": "./.scripts/healthcheck.sh",
		"dip721:deploy": "./.scripts/deploy.sh",
```